packages/astro/src/integration/index.ts

```diff
@@ -79,3 +84,17 @@ function findDefaultSdkInitFile(type: 'server' | 'client'): string | undefined {
     .map(ext => path.resolve(path.join(process.cwd(), `sentry.${type}.config.${ext}`)))
     .find(filename => fs.existsSync(filename));
 }
+
+function getSourcemapsAssetsGlob(config: AstroConfig): string {
+  // outDir is stored as a `file://` URL
+  const outDirPathname = config.outDir && config.outDir.pathname;
+  const rootDirName = config.root && config.root.pathname;
```

Comment:
would the following make sense here

```suggestion
  const rootDirName = path.resolve(config.root && config.root.pathname);
```

to fall back to cwd if the root is not defined?

Reply:
yeah we could do this but technically they should both always be available